lib/yext/api/utils/params.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Yext
+  module Api
+    module Utils
+      # A utility class for performing normal modifications to webhooks to convert/normalize data
+      # before passing it along so that the receiver doesn't have to worry/thnk about it.
```

Comment:
Should standardize on naming. Is it a processor? a receiver? a listener?

Reply:
That is probably a lost cause with me.  I am a name idiot.